README.md

```diff
@@ -157,402 +177,393 @@ cd tdx/guest-tools
-You didn't set Intel PCS API key. You can set it later in config/default.json.  
-Choose caching fill method : [LAZY] (LAZY/OFFLINE/REQ) :
-Set PCCS server administrator password: <PCCS-ADMIN-PASSWORD>
-Re-enter administrator password: <PCCS-ADMIN-PASSWORD>
-Set PCCS server user password: <PCCS-SERVER-USER-PASSWORD>
-Re-enter user password: <PCCS-SERVER-USER-PASSWORD>
-Do you want to generate insecure HTTPS key and cert for PCCS service? [Y] (Y/N) :N
-```
+	```bash
+	sudo systemctl status mpa_registration_tool
+	```
 
-7. Restart the PCCS service.
+	An example output:
 
-```bash
-sudo systemctl restart pccs
-```
+	```bash
+	mpa_registration_tool.service - Intel MPA Registration
+		Loaded: loaded (/usr/lib/systemd/system/mpa_registration_tool.service; enabled; preset: enabled)
+		Active: inactive (dead) since Tue 2024-04-09 22:54:50 UTC; 11h ago
+	Duration: 46ms
+	Main PID: 3409 (code=exited, status=0/SUCCESS)
+			CPU: 21ms
 
-8. Verify the PCCS service is running properly.
+	Apr 09 22:54:50 right-glider-515046 systemd[1]: Started mpa_registration_tool.service - Intel MPA Registratio>
+	Apr 09 22:54:50 right-glider-515046 systemd[1]: mpa_registration_tool.service: Deactivated successfully.
+	```
 
-```bash
-sudo systemctl status pccs
-```
-9. Platform registration.
+	If the failure occurred, you might want to boot into the BIOS and perform an Intel SGX Factory reset.
+	This can be done by selecting `SGX Factory Reset` in `Socket Configuration > Processor Configuration`.
 
-The platform registration is done with `mpa_registration_tool`, the service is run on system start up,
-registers the platform and gets deactivated. Please check the service does not output any error message:
 
-```bash
-sudo systemctl status mpa_registration_tool
-```
+### 8.3 Setup [Intel Tiber Trust Services CLI](https://github.com/intel/trustauthority-client-for-go) inside TD
 
-```bash
-mpa_registration_tool.service - Intel MPA Registration
-     Loaded: loaded (/usr/lib/systemd/system/mpa_registration_tool.service; enabled; preset: enabled)
-     Active: inactive (dead) since Tue 2024-04-09 22:54:50 UTC; 11h ago
-   Duration: 46ms
-   Main PID: 3409 (code=exited, status=0/SUCCESS)
-        CPU: 21ms
-
-Apr 09 22:54:50 right-glider-515046 systemd[1]: Started mpa_registration_tool.service - Intel MPA Registratio>
-Apr 09 22:54:50 right-glider-515046 systemd[1]: mpa_registration_tool.service: Deactivated successfully.
-```
+NOTE: If you have already installed the attestation components as part of the TD image creation,
+you proceed to [step 4](#verify-itts-client-version).
 
-If the failure occurred, you might want to boot into the BIOS and perform `SGX Factory Reset` (go to `Socket Configuration > Processor Configuration`).
+1. [Boot a TD](#boot-td) and connect to it.
 
-### Setup [Intel Trust Authority (ITA) Client](https://github.com/intel/trustauthority-client-for-go) on Guest 
+2. Clone this repository:
```

Comment:
For consistency with other parts of the README, I would suggest the following wording rather than instructing a user to clone from a specific branch:

Download this repository by downloading an asset file from the [releases page on GitHub](https://github.com/canonical/tdx/releases) or by cloning the repository at the appropriate tag.